content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -82,6 +82,8 @@ that was just configured in the :guilabel:`Create invoices` pop-up form appears
    If :guilabel:`Based on Delivered Quantity (Manual)` is chosen as the :guilabel:`Invoicing
    Policy`, an invoice will **not** be able to be created.
 
+.. _sales/invoicing/down_payment/50-percent-down-payments:
```

Comment:
Small update to only include the parent folder in the custom anchor convention: 
`app or module name/(optional parent folder)/section heading`
```suggestion
.. _sales/invoicing/50-percent-down-payments:
```